init_env.sh

```diff
@@ -1,15 +1,15 @@
 #!/bin/bash
-MULTILINE1=$(conda env list | grep -F substorm-nlp)
+MULTILINE1=$(conda env list | grep -F rpa-tomorrow)
 
 if [ "$MULTILINE1" = "" ]
 then
     echo "no conda env found, creating env..."
-    conda env create -f substorm-nlp.yml
+    conda env create -f rpa-tomorrow.yml
 fi
 
-echo "activate conda substorm-nlp env..."
+echo "activate conda RPA Tomorrow env..."
```

Comment:
hmm, why is this named differently?
```suggestion
echo "activate conda rpa-tomorrow env..."
```

Reply:
Ok fixed now!